packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -33,6 +33,11 @@ export interface OrganizedParticipantsResult {
 }
 
 const DEFAULT_MAX_OVERFLOW_GALLERY_DOMINANT_SPEAKERS = 6;
+const DEFAULT_MAX_VIDEO_SREAMS = 4;
+/* @conditional-compile-remove(large-gallery) */
+const MAX_VIDEO_STREAMS_FOR_LARGE_GALLERY = 49;
```

Comment:
should this be 48? (if you (local) is one of them?)

Reply:
good catch!